src/apps/api/tests/test_queue_api_permissions.py

```diff
@@ -104,3 +99,42 @@ def test_organizer_user_cannot_delete_or_edit_queue(self):
         )
         assert response.status_code != 204
         assert Queue.objects.filter(pk=self.created_queue.id).exists()
+
+    # Test user queue limits
+    def test_queue_limits_for_regular_users(self):
+        [QueueFactory(owner=self.user) for _ in range(3)]
+
+        self.client.login(username='test', password='test')
+        data = self._get_test_update_data(return_id=False)
+        # default limit is 5
```

Comment:
add
`assert self.user.<whatever the path to queue limit is> == 5`
so that this test is more secure.

... or ... you could change the rabbit queue limit to be < 5, then you wouldn't need to do the `QueueFactory() for _ in range()` bit. Either way, we shouldn't build a test around a default value, since maybe that will change, and break our test.

Reply:
Changed to just lower user limit.
✔️